sources/gov/kaunas/municipality/road_signs.yaml

```diff
@@ -0,0 +1,94 @@
+id: "gov/kaunas/municipality/road_signs"
+title: "Kaunas road signs"
+description: "CSV file containing data about Kaunas road signs"
+type: "source"
+source:
+  - "https://github.com/Kauno-miestas/Kauno-Planas/blob/master/Transportas/csv/Zenklai.csv"
+  - "csv:"
+provider: "gov/kaunas/municipality"
+objects:
+  road_signs:
+    source: https://github.com/Kauno-miestas/Kauno-Planas/blob/master/Transportas/csv/Zenklai.csv
```

Comment:
I think, it would be nice to specify parameters how to parse `.csv` file. Probably something like this:

```yaml
source:
  - "https://github.com/Kauno-miestas/Kauno-Planas/blob/master/Transportas/csv/Zenklai.csv"
  - type: "csv"
    delim: ","
```

I will try to improve validation script, so that it could actually try to read the source.